### ISSUE TITLE
Info: Pressure Advance on Bowden Extruder ( Ender 3)

### ISSUE THREAD

**Issue by muenznej** [KevinOConnor/klipper#1025]
Hello, I just wanted to contribute some findings regarding my bowden extruder setup in case someone is having problems with it. My printer is a Ender 3 and i replaced the stock hotend for a e3d volcano clone (in order to realize a appropriate flow rate ( up to 30-40 mm3/s). So that helped with faster printing already. The next thing that seems to help is posted in issue #516 :

> Not about klipper but they is a mechanical solution to reduce the accumulation of pressure in the bowden tube. Check this out https://www.facebook.com/groups/Hypercube.Evo/permalink/192106034761003/
> For me it made a really huge difference.

And i agree that did make a difference for me. I used a Capricorn clone of unknown quality but could reduce the pressure_advance setting to ~0.3 (with the overall upgrades)  (it wasn't working (at all?) even with > 1.0 before)
I also added this glas fiber tape to lock the tube to the extruder setup and reduced the possibility of movement and compression. I think the next cheap upgrade improving this setup would be better fittings to hold the tube tight. Any recommendations? 
Its not perfect and not optimized but its the first time PA actually does improve (at least the square-test)

just my 2 cents for anyone who has/had similar challenges.

Btw is there another better(?) place to talk about klipper in conjunction with fast printing?

**Comment by KevinOConnor** [comment]
FWIW, I got a chance to try the fiber glass tape trick on the ptfe tube of my bowden extruder.  Alas, it didn't improve the pressure advance settings for me.  (This is a delta printer with an ~500mm bowden - the ptfe tubing is some generic brand but seems pretty rigid.)

-Kevin

**Comment by welo78** [comment]
@muenznej Unfotunatelly I currently have the same issue as you have. I just changed the stock extruder to an E3D V6 Volcano clone with a Titan clone since I was having issues with filament not having enough space/time to heat up properly. Printing PLA at 230 to get that 100mm/s speeds is not a good idea, specially for PLA. 

I am now currently trying this setup since regular mk9 extruder and ender 3 hotend did not perform at all with pressure advance on. Missed steps on PA set to 0.8 or higher with no visible change to corners. I since then tried capricorn tubing and other stuff suggested by community with no change. Last thing I read was to try to disable retractions and lower the accelerations so that PA can have time to actually do something.

**Comment by muenznej** [comment]
> FWIW, I got a chance to try the fiber glass tape trick on the ptfe tube of my bowden extruder. Alas, it didn't improve the pressure advance settings for me. (This is a delta printer with an ~500mm bowden - the ptfe tubing is some generic brand but seems pretty rigid.)
> 
> -Kevin

Thx for testing it yourself. As mentioned I have changed a big chunk of my setup and i think the most relevant part for getting the improved corners  was the higher melting power of the hotend. Hence i am a bit intrigued why @welo78  does not see any improvements. I have retractions below 1mm in addition to the PA of about .4 (now). I am not saying its perfect, but its a huge improvement for corners. The biggest downside is the increased oozing of the volcano setup. I am changing to a geared extruder and report back if it works better/good with the fast retraction acceleration.
Regarding the PTFE bowden tube itself I think its clear, that there is still a lot of friction since the tube is pulled into the couplers when strong retractions are carried out. I tried to fix this by taping everything more rigidly, but it didnt change much.
Unfortunately I have little spare time right now, so I wont be able to make a huge test series any time soon :/

P.S.: I think volcano and direct drive titan setup will be the best final setup.
